src/applications/caregivers/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import moment from 'moment';
+import { connect } from 'react-redux';
+import Scroll from 'react-scroll';
+
+import { focusElement } from 'platform/utilities/ui';
+
+const scroller = Scroll.scroller;
+const scrollToTop = () => {
+  scroller.scrollTo('topScrollElement', {
+    duration: 500,
+    delay: 0,
+    smooth: true,
+  });
+};
+
+export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    focusElement('.schemaform-title > h1');
+    scrollToTop();
```

Comment:
I believe we have a utility function for this [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/utilities/ui/scrollToTop.js) if you want to use it instead of creating your own `scrollToTop` func.